### ISSUE TITLE
is_stop seems to be broken for german language

### ISSUE THREAD

**Issue by sbl** [explosion/spaCy#729]
It seems that the german `is_stop` method on `Token` is not working, unless I'm using the API wrong. The `en` version is working as expected as shown below.

## reproduce

```python
import spacy
nlp = spacy.load('de')

doc = nlp('bald bei beide beiden beim beispiel bekannt bereits besonders besser besten bin')
for t in doc:
    print(t, t.is_stop)
'''
output:
>> bald False
>> bei False
>> beide False
>> beiden False
>> beim False
>> ...
'''

# en behaviour
nlp = spacy.load('en')
doc = nlp('a about above across after afterwards again against all almost alone along')

for t in doc:
    print(t, t.is_stop)
'''
output:

>> a True
>> about True
>> above True
>> across True
>> after True
>> ...
'''
```

## Your Environment
* Operating System: mac os sierra
* Python Version Used: python 3.6.0
* spaCy Version Used: 1.50
* Environment Information:

**Comment by honnibal** [comment]
Unfortunately while this is fixed in the code, the pre-trained German model comes with the old value of this feature cached. We need to publish a new model to address this.

In the meantime, you can work around the problem by writing to the data as follows:

```python

import spacy
nlp = spacy.load('de')

for word in nlp.Defaults.stop_words:
    lex = nlp.vocab[word]
    lex.is_stop = True
```

Text you parse with the modified vocabulary should then reflect the updated stop-words data.

**Comment by sbl** [comment]
Thanks for the info. I'll try to workaround this for now.

**Comment by ines** [comment]
Fixed in the latest update, `v2.0.0a18`. There's also a full `de_core_web_sm` model now – [see here](https://alpha.spacy.io/models/de#de_core_news_sm) for details and accuracy.
